pkgs/tools/security/pcsclite/default.nix

```
@@ -1,4 +1,15 @@
-{ lib, stdenv, fetchurl, pkg-config, udev, dbus, perl, python3, IOKit }:
+{ stdenv
+, lib
+, fetchurl
+, autoreconfHook
+, pkg-config
+, perl
+, python3
+, dbus
+, polkit ? null
+, systemd ? null
+, IOKit ? null
```

```suggestion
, polkit
, systemd
, IOKit
```

If we get any error here than we did something wrong and darwin will be broken.